synfig-studio/src/gui/trees/layertree.cpp

```diff
@@ -105,8 +105,8 @@ using namespace studio;
 bool LayerTree::onKeyPress(GdkEventKey* event)
 {
 
-	switch (event->hardware_keycode) {
-		case 119: { // 'Del' key pressed
+	switch (event->keyval) {
+		case GDK_KEY_Delete: { // 'Del' key pressed
 			std::cout << "LayerTree::onKeyPress: Got delete!" << std::endl;
```

Comment:
I suggest to remove this line. ^__^

Reply:
Really? :/ It took me a long time to track where/if delete was used due to non-enum key usage.
Same about method name not being "on_key_press_down()".
I read `git blame` and this code piece was written only in commits 27966f3919 and 65643eee77 .
Apparently, there is no particular reason to avoid `keyval` and `GDK_KEY_*` enums.
Should I really remove those 3 lines (the one following too).

Reply:
I meant only this line ^__^ -
```
std::cout << "LayerTree::onKeyPress: Got delete!" << std::endl;
```
(the one which produces debug output to console).

Reply:
Oh. Done :)